superset/key_value/commands/args.py

```diff
@@ -0,0 +1,29 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from typing import Dict
+
+from flask_appbuilder.security.sqla.models import User
+from typing_extensions import TypedDict
+
+
+class Args(TypedDict, total=False):
```

Comment:
Could we make the name more explicit, like `KeyValueArgs`? Also, usually `args` refers to a list of values, while `kwargs` refers to a dictionary, so maybe `Args` isn't the best name for this, as it's maybe more like an "entry", which makes it slightly ambiguous with the `Entry` class. So maybe we could make these more unambiguous and clarify their relative roles?

Reply:
They are more related to command parameters than arguments for the key-value store. I renamed it to `CommandParameters` 😄